rust/rsc/src/common/database.rs

```diff
@@ -1,17 +1,18 @@
 use chrono::NaiveDateTime;
 use data_encoding::BASE64;
 use entity::prelude::{
-    Blob, BlobStore, LocalBlobStore, OutputDir, OutputFile, OutputSymlink, VisibleFile,
+    Blob, BlobStore, Job, LocalBlobStore, OutputDir, OutputFile, OutputSymlink, VisibleFile,
 };
 use entity::{
     api_key, blob, blob_store, job, local_blob_store, output_dir, output_file, output_symlink,
     visible_file,
 };
 use itertools::Itertools;
 use rand_core::{OsRng, RngCore};
+use sea_orm::ExecResult;
 use sea_orm::{
     prelude::Uuid, ActiveModelTrait, ActiveValue::*, ColumnTrait, ConnectionTrait, DbBackend,
-    DbErr, DeleteResult, EntityTrait, QueryFilter, Statement,
+    DbErr, DeleteResult, EntityTrait, PaginatorTrait, QueryFilter, QuerySelect, Statement,
```

Comment:
Does `Paginator` get used when you do the `LIMIT`? I would have expected to see some `page` function if we are using this trait. But I've been assuming there is some "unused import" check that would catch things like that.

Reply:
imo Rust traits lean from kind of confusing to very confusing. This one is on the very side.

The function `.count(db)` from `let total = entity::job::Entity::find().count(db).await?;` is provided by the `PaginatorTrait` which is why it got pulled in

Reply:
> But I've been assuming there is some "unused import" check that would catch things like that.

This is correct to assume :)